test/functional/Deal/HubSpotDealClientFunctionalTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using RapidCore.Network;
+using Skarp.HubSpotClient.Deal;
+using Skarp.HubSpotClient.Deal.Dto;
+using Skarp.HubSpotClient.Core.Requests;
+using Xunit;
+using Xunit.Abstractions;
+using Skarp.HubSpotClient.FunctionalTests.Mocks.Deal;
+
+namespace Skarp.HubSpotClient.FunctionalTests.Deal
+{
+    public class HubSpotDealClientFunctionalTest : FunctionalTestBase<HubSpotDealClient>
```

Comment:
nice with a functional test for it all :100:

Reply:
Updated the project dependency - never did that addition intentionally 👍

Reply:
@JoachimS Great thanks - and I know :) 👍 
Thanks for the PR.